src/services/user/services/AdminUsers.js

```diff
@@ -182,17 +182,29 @@ class AdminUsers {
 		const params = await this.prepareParams(_id, _params);
 		await this.checkIfExternallyManaged(params.query.schoolId);
 
-		const { email } = _data;
-		await this.checkMail(email, _id);
-		await this.updateAccount(email, _id);
+		const userData = _data;
+		await this.checkMail(userData.email, _id);
+		if (!userData.createAccount) {
```

Comment:
Why rename `_data`? We can destrucure it with `const { email, createAccount } = _data` or use `_data` instead of `userData`.

Reply:
Fixed

Reply:
Fixed